src/bin/common/parse_command.ml

```diff
@@ -1238,8 +1304,16 @@ let parse_fmt_opt =
     Arg.(value & opt formatter_conv Stderr & info ["err-formatter"] ~docs ~doc)
   in
 
+  let mdl_formatter =
+    let doc = Format.sprintf
+        "Set the model formatter used by default to output model and
+         interpretation. Possible values are %s."
+        (Arg.doc_alts ["stdout"; "stderr"; "<filename>"]) in
+    Arg.(value & opt formatter_conv Stdout & info ["mdl-formatter"] ~docs ~doc)
```

Comment:
The name `formatter` is meaningless for users of alt-ergo (it's only pertinent for those that program in ocaml). Something like `model-output` would be clearer (both for the doc string and the option name).